cmd/agent/main_windows.go

```diff
@@ -39,9 +39,10 @@ func init() {
 	fmt.Printf("main_windows.init()")
 	pd, err := winutil.GetProgramDataDir()
 	if err == nil {
-		defaultConfigPath = filepath.Join(pd, "Datadog", "datadog.yaml")
-		defaultConfdPath = filepath.Join(pd, "Datadog", "conf.d")
-		defaultLogFilePath = filepath.Join(pd, "Datadog", "logs", "process.log")
+		defaultConfigPath = filepath.Join(pd, "datadog.yaml")
+		defaultOldConfigPath = filepath.Join(pd, "datadog.conf")
```

Comment:
I think you don't have this `defaultOldConfigPath` initialized properly. All other defaultXXX variables have initializations several lines above, so should be `defaultOldConfigPath`.

Reply:
it looks like at some point the `defaultOldConfigPath` was removed completely?